src/main/java/core/basesyntax/services/ReadService.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.services;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class ReadService {
```

Comment:
```suggestion
public class FileReaderImpl implements FileReader {
```
lets create an interface (interface segregation)